src/capturer/cosmos/cosmos.ts

```diff
@@ -20,6 +20,13 @@ export const getChangeFeedIteratorOptions = (
   maxItemCount: maxItemCount && maxItemCount > 0 ? maxItemCount : 1,
 });
 
+// eslint-disable-next-line no-var
+var shouldExitExternal = false;
+
+export const setShouldExitExternal = (value: boolean): void => {
```

Comment:
what's this for? I'd add a comment

Reply:
It's an external signal to exit from the loop that is waiting for database changes